dff/core/engine/conditions.py

```diff
@@ -169,21 +171,18 @@ def has_last_labels(
     **kwargs,
 ) -> Callable:
```

Comment:
Should we add Callable signature (☝️): `Callable[[Context, Actor, Any, Any], bool]` (AFAIK) here.